ss/ss.go

```diff
@@ -49,17 +40,17 @@ func CreateComDetailsFromNode(cs *client.ClientSet, node *corev1.Node, tcpFile,
 		}
 	}()
 
-	ssOutTCP, err := debugPod.ExecWithRetry("ss -anplt", interval, duration)
+	ssOutTCP, err := debugPod.ExecWithRetry("ss -anpltH", interval, duration)
 	if err != nil {
 		return nil, err
 	}
-	ssOutUDP, err := debugPod.ExecWithRetry("ss -anplu", interval, duration)
+	ssOutUDP, err := debugPod.ExecWithRetry("ss -anpluH", interval, duration)
 	if err != nil {
 		return nil, err
 	}
 
-	ssOutFilteredTCP := filterStrings(tcpSSFilterFn, splitByLines(ssOutTCP))
-	ssOutFilteredUDP := filterStrings(udpSSFilterFn, splitByLines(ssOutUDP))
+	ssOutFilteredTCP := filterStrings(filterOutFn, splitByLines(ssOutTCP))
+	ssOutFilteredUDP := filterStrings(filterOutFn, splitByLines(ssOutUDP))
```

Comment:
nit: since both use the same filter, and filterStrings is only called here we can just rename filterStrings to something more meaningful and remove the filterFn semantics